internal/cli/atlas/deployments/search/indexes/create.go

```diff
@@ -78,7 +81,12 @@ func (opts *CreateOpts) Run(ctx context.Context) error {
 		return err
 	}
 
-	opts.index, err = opts.mongodbClient.Database(opts.index.Database).CreateSearchIndex(ctx, opts.index.CollectionName, opts.index)
+	db := opts.mongodbClient.Database(opts.index.Database)
+	if idx, _ := db.SearchIndexByName(ctx, opts.index.Name, opts.index.CollectionName); idx != nil {
+		return ErrSearchIndexDuplicated
+	}
+
+	opts.index, err = db.CreateSearchIndex(ctx, opts.index.CollectionName, opts.index)
 	if err != nil {
 		return err
 	}
```

Comment:
[optional]
```
	if err != nil {
		return err
	}
	return nil
```
replace it with `return err`

Reply:
will do that in my next PR